{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cna@vuldb.com",
			"ID": "CVE-2022-4526"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "A vulnerability was found in django-photologue up to 3.15.1 and classified as problematic. Affected by this issue is some unknown functionality of the file photologue/templates/photologue/photo_detail.html of the component Default Template Handler. The manipulation of the argument object.caption leads to cross site scripting. The attack may be launched remotely. Upgrading to version 3.16 is able to address this issue. The name of the patch is 960cb060ce5e2964e6d716ff787c72fc18a371e7. It is recommended to apply a patch to fix this issue. VDB-215906 is the identifier assigned to this vulnerability."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-79"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://github.com/richardbarran/django-photologue/commit/960cb060ce5e2964e6d716ff787c72fc18a371e7",
					"tags": [
						"Patch",
						"Third Party Advisory"
					],
					"url": "https://github.com/richardbarran/django-photologue/commit/960cb060ce5e2964e6d716ff787c72fc18a371e7"
				},
				{
					"name": "https://github.com/richardbarran/django-photologue/commit/960cb060ce5e2964e6d716ff787c72fc18a371e7",
					"tags": [
						"Patch",
						"Third Party Advisory"
					],
					"url": "https://github.com/richardbarran/django-photologue/commit/960cb060ce5e2964e6d716ff787c72fc18a371e7"
				},
				{
					"name": "https://github.com/richardbarran/django-photologue/issues/223",
					"tags": [
						"Issue Tracking",
						"Third Party Advisory"
					],
					"url": "https://github.com/richardbarran/django-photologue/issues/223"
				},
				{
					"name": "https://github.com/richardbarran/django-photologue/issues/223",
					"tags": [
						"Issue Tracking",
						"Third Party Advisory"
					],
					"url": "https://github.com/richardbarran/django-photologue/issues/223"
				},
				{
					"name": "https://vuldb.com/?id.215906",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://vuldb.com/?id.215906"
				},
				{
					"name": "https://vuldb.com/?id.215906",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://vuldb.com/?id.215906"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "NETWORK",
				"availabilityImpact": "NONE",
				"baseScore": 6.1,
				"baseSeverity": "MEDIUM",
				"confidentialityImpact": "LOW",
				"integrityImpact": "LOW",
				"privilegesRequired": "NONE",
				"scope": "CHANGED",
				"userInteraction": "REQUIRED",
				"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
				"version": "3.1"
			},
			"exploitabilityScore": 2.8,
			"impactScore": 2.7
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:django-photologue_project:django-photologue:*:*:*:*:*:*:*:*",
						"versionEndIncluding": "3.15.1",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-11-21T07:35Z",
	"publishedDate": "2022-12-15T21:15Z"
}
